src/utils/viewUtils.ts

```diff
@@ -1,14 +1,14 @@
-import { addMinutes, format, formatDistanceToNow, parseISO } from 'date-fns';
+import { addMinutes, format, formatDistanceToNowStrict, parseISO } from 'date-fns';
 
 const getIsoToLocalDate = (iso: string): Date => {
   const date = parseISO(iso);
   return addMinutes(date, -date.getTimezoneOffset());
 };
 
 export const getDistanteToNow = (iso: string): string => {
-  return formatDistanceToNow(getIsoToLocalDate(iso), { addSuffix: true });
+  return formatDistanceToNowStrict(getIsoToLocalDate(iso), { addSuffix: true });
 };
 
 export const getFormattedDate = (iso: string, formatting?: string): string => {
```

Comment:
![image](https://user-images.githubusercontent.com/66584472/109317432-0d60bf00-7890-11eb-9fca-1d99520a50d3.png)

Back end is sending logs with timestamps in the format `dd/MM/yyyy HH:mm:ss`.
That's causing `getIsoToLocalDate` to return an `Invalid Date`.
Will fix ASAP on back end.

In the meantime, I'll also add an issue for improving the error handling in case of bad data.